### PR TITLE
Code linters 416

### LINKED ISSUE #416: Find formatters and linters for JS and C++
We need them to enforce code formatting and to check code quality.

### PR DESCRIPTION
**Pull request by VimFreeman**
# Assignee Section

## Description
Changed the README file to include instructions as to how to set up VS Code.
Changed various files used by the formatters and VS Code

### Steps for Testing
Verify that the plugins are appropriate and that their configs are adequate:
- standardjs (no config)
- clangd (.clang-format)
- Python (.yapf    .pylintrc)

Verify that all atom packages are relevant and use the appropriate configs..
Look in `.atom/package-list.txt` for the package names
and `.atom/config.cson` for the custom configs

closes #416 

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.

- [ ] Local Test Performed Successfully
- [ ] [General Integration Test](https://docs.google.com/document/d/1ug0CpA1cIzURP8DDFSvCt2CEJJSwJ6Ta6B1LG_hYk6I/edit) Performed Successfully

For Pull Requests that do not include code changes, it is not required to perform the tests above.

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+Language: Cpp
+BasedOnStyle: Google
```
Comment (MartensCedric) [inline .clang-format:3]: image

What would be best for embedded C++ @PeterGhimself ?

Comment (MartensCedric) [reply .clang-format:3]: can we add new formats?

**Review by MartensCedric — DISMISSED** [review]

File: .clang-format (hunk 1 above)
Comment (AFknUsername) [reply .clang-format:3]: We can create our own set of rules using the 1187729484 different options

**Comment by VimFreeman** [comment]
Closing cause kinda useless

**Pull request closed without merging**